lib/neo4j/shared/type_converters.rb

```diff
@@ -102,7 +108,7 @@ def converted_values
         end
 
         def db_type
-          ActiveAttr::Typecasting::Boolean
+          Neo4j::Shared::Boolean
```

Comment:
Using constants might keep some of the code a bit simpler where type conversion is concerned. It might be completely necessary or at least significantly easier to use them than symbols with QueryProxy, too, but I can't remember what all that code looks like offhand.